_data/tools.yml

```diff
@@ -421,6 +421,15 @@
 
 # Server implementations
 
+- name: tsoa
+  category: server
```

Comment:
Is it potentially `data-validation` as that is where we put other request validators.

Reply:
Thank you for pointing that out. The tsoa library does provide `data-validation` and it is also within the `server` category. I've updated the PR to include both categories.